README.md

```diff
@@ -1,6 +1,8 @@
 # Simple Calculator
 <img alt="Logo" src="app/src/main/res/mipmap-xxxhdpi/ic_launcher.png" width="80">
 
+[![Build Status](https://travis-ci.org/jusleg/simple-calculator.svg?branch=master)](https://travis-ci.org/jusleg/simple-calculator) [![codebeat badge](https://codebeat.co/badges/9ce2c059-5bb7-46bd-b512-a746ce275690)](https://codebeat.co/projects/github-com-jusleg-simple-calculator-master)
+
```

Comment:
Badges are neato

Reply:
I found those two badges would help track the quality of our code. We should look into codebeat to find things to fix

Reply:
I strongly agree. Thank you kindly.